visualization/index.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express')
 const app = express()
-const port = 3001
-const { getCultivarSites, getExperimentSites, getTreatmentSites } = require('./pg_joins')
+const port = 8075
+const host= '0.0.0.0'
```

Comment:
See docker compose comment above: potentially a good place for environment variables

Reply:
Good idea, new issue #72